hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieAppendHandle.java

```diff
@@ -329,7 +329,7 @@ private MetadataValues populateMetadataFields(HoodieRecord<T> hoodieRecord) {
     if (config.populateMetaFields()) {
       String seqId =
           HoodieRecord.generateSequenceId(instantTime, getPartitionId(), RECORD_COUNTER.getAndIncrement());
-      metadataValues.setFileName(fileId);
+      metadataValues.setFileName(writer.getLogFile().getFileName());
```

Comment:
This is a breaking change on the storage format where the `_hoodie_file_name` stores the file name instead of the file ID in log files.  Blocking the PR from merging until we have an alignment.

Reply:
> This is a breaking change on the storage format where the `_hoodie_file_name` stores the file name instead of the file ID in log files. Blocking the PR from merging until we have an alignment.

Agreed. Waiting for the result of your discussion